bijou.js

```diff
@@ -55,6 +55,8 @@ Contributors to Bijou.js:
 ╠═══════════════════╬════════════════════════════╣
 ║ YOYITsM3M8        ║ Suggested a lot of         ║
 ║ (AKA retronbv)    ║ features and bug fixes.    ║
+║═══════════════════║════════════════════════════║
+║ thecoder876       ║ Made some improvements.     ║
```

Comment:
Try being more specific?

Reply:
Seriously, none of the other ones are very specific. For example, compare `Suggested a lot of features and bug fixes.` with `Made some improvements`. What's the difference?

Reply:
@D1WK Ok, I fixed it.

Reply:
@thecoder876  
> Seriously, none of the other ones are very specific. For example, compare Suggested a lot of features and bug fixes

:(